examples/ruby/README.md

```diff
@@ -1,3 +1,7 @@
+**API `v1` is deprecated**
+
+***
+
```

Comment:
Ahh, the readme just needs to be updated. The API wrapper doesn't care about the data structure though.